src/resources/Catalogs/Catalog.ts

```diff
@@ -22,8 +22,10 @@ export default class Catalog extends Resource {
         return this.api.get<CatalogModel>(`${Catalog.baseUrl}/${catalogId}`);
     }
 
-    getFields(catalogId: string) {
-        return this.api.get<CachedCatalogFieldsModel>(`${Catalog.baseUrl}/${catalogId}/fields`);
+    getFields(catalogId: string, bypassCache?: boolean) {
```

Comment:
What about exposing a `CatalogFieldsOptions`?

Reply:
I like this idea! I'll implement that instead. 

I was actually hesitating between a params object or a simple parameter like in this solution. :sweat_smile:

Reply:
All good!